File: Backend/ECORUTAAS/src/main/java/com/cibertec/models/Estadistica.java
package com.cibertec.models;



import jakarta.persistence.*;
import lombok.*;
import java.time.LocalDate;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "estadistica")
public class Estadistica {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id_estadistica;

    private LocalDate fecha;
    private Integer total_usuarios;
    private Integer total_rutas;
    private Integer total_alquileres;
    private Double co2_ahorrado;
}

File: Mobile/app/src/main/java/com/cibertec/proyectoecorutasapp/models/Estadistica.kt
package com.cibertec.proyectoecorutasapp.models

data class Estadistica(
    val id_estadistica: Int? = null,
    val fecha: String,
    val total_usuarios: Int,
    val total_rutas: Int,
    val total_alquileres: Int,
    val co2_ahorrado: Double
)
